assembl/graphql/schema.py

```diff
@@ -82,6 +118,62 @@ def langstring_from_input_entries(entries):
     return None
 
 
+def update_langstring_from_input_entries(obj, attr, entries):
+    """Update langstring from getattr(obj, attr) based on GraphQL LangStringEntryInput entries.
+    """
+    langstring = getattr(obj, attr, None)
+    if langstring is None:
+        new_langstring = langstring_from_input_entries(entries)
```

Comment:
The first value is not special at all, it's just the way the langstring api works.